### ISSUE TITLE
Optical Flow Labels

### ISSUE THREAD

**Issue by niklasAust** [hzwer/arXiv2020-RIFE#114]
Good evening!

Can you tell me how you generated the optical flow labels in the 100 sample subset of the vimeo-90k dataset?

I cannot reproduce these in the same way with [pytorch-liteflownet](https://github.com/sniklaus/pytorch-liteflownet)
This is the one I can generate with liteflownet

`python run.py --model default --first ./images/first.png --second ./images/second.png --out ./out.flo
`

![LiteFlowNet](https://user-images.githubusercontent.com/6423533/109069621-c341d700-76f1-11eb-9fd1-2befe1e48288.png)

and this is the one that is used in the trainings sample net
![RIFE-TrainingFlow](https://user-images.githubusercontent.com/6423533/109069206-33039200-76f1-11eb-8243-c573668252e3.png)

)

The optical flow I get is much more blurry.
Did you use special code or a different model than the pretrained ones?
I already used different Input image sizes.
Rescaled from 1x to 5x, but all are blurry.

Thank you in advance!

**Comment by hzwer** [comment]
Hello,
        I finetune the liteflownet on Vimeo90K for 1 epoch. That finetune can sharpen the optical flow labels. However, my experiment show that sharper optical flow labels can just improve final model performance very slightly. 
I will update my optical flow examples later to prevent misunderstanding.

**Comment by niklasAust** [comment]
Thank you for the clarification!
